README.md

```diff
@@ -1 +1,56 @@
-# Getting started with Cisco IOS-XE Smart licensing APIs
\ No newline at end of file
+# Getting Started with Cisco IOS-XE Smart licensing APIs
+
+Sample code in Python that shows how to call IOS-XE SLP REST APIs. You can automate the usage reporting workflows using these APIs.
+
+For information about Smart Licensing Using Policy (SLP), see [About Smart Licensing Using Policy](ios-xe-smart-licensing-api/intro.md#about-smart-licensing-using-policy).
+
+## Use Case Description
+
+For information about the purpose of each currently available API, see [About IOS-XE SLP REST APIs](ios-xe-smart-licensing-api/intro.md#about-ios-xe-slp-rest-apis).
+
+Future enhancements include an API to return a Smart Licensing Authorization Code (SLAC).
+
+## Installation
+
+Python must be installed. Further the following Python packages are required:
+* os
+* shutil
+* json
+* xmltodict
+* requests
+* yaml
+* tarfile
+
+## Configuration
+Configure the following where? we have to check CSSM API docs to know this and then add. 
```

Comment:
Configure the following in the "config.yaml" file.

Reply:
I've made the update for the "API keys" and "devices" sections.  
The part about CSSM APIs ...
1. If they are not available I think we should stop at saying just that.  (instead of providing info about the docs etc)
2. We've referred to the option of CSSM APIs in several places. I will need to update all instances.